schemas/input/csl-data.json

```
@@ -4,425 +4,477 @@
+          }
+        },
+        "chair": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "collection-editor": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "compiler": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "composer": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "container-author": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "contributor": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "curator": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "director": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "editor": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "editorial-director": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "executive-producer": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "interviewer": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "illustrator": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "organizer": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "original-author": {
+          "description": "[Deprecated - Use `author` on the `original` related item instead]",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "performer": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "producer": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "recipient": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/name-variable"
+          }
+        },
+        "reviewed-author": {
+          "description": "[Deprecated - Use `author` on the `reviewed` related item instead]",
```

```suggestion
          "description": "Can also be be entered as `author` on the `reviewed` related item",
```

I think we could leave these `reviewed-*` variables available for direct entry into item data.

---

Hmm I think we should take the opportunity to simplify the schema and remove redundant fields.

---

My major concerns are always:
  1. Backwards compatibility with existing CSL-JSON data in the wild (especially embedded in word processor documents) (e.g., why `journalAbbreviation` and `shortTitle` stick around)
  2. Implementation ease in applications; I don't want breakage around the existing `reviewed-author` structure in Zotero, for example, to get in the way of it adopting CSL 1.1 entirely.
  3. Ease of use for entering flat key-value pairs when users are entering information manually (e.g., in Mendeley's Note field).

---

But keep in mind I didn't remove these; I marked them deprecated, where the audience is mostly developers.

We could hold off on removing to 1.2. or 1.3; even later if needed be.

But saying nothing will enshrine the status quo for a long time.